test/functional/forecast.test.ts

```diff
@@ -26,6 +26,25 @@ describe('Beach forecast functional tests', () => {
     await new Beach(defaultBeach).save();
     token = AuthService.generateToken(user.toJSON());
   });
+  // No cache to this test
+  it('should return 500 if something goes wrong during the processing', async () => {
```

Comment:
what about we clean all the caching data on the `beforeEach`?

Reply:
That's nice approach!